### DISCUSSION TITLE
Why we can't get input parameter by task(task_id)

### DISCUSSION THREAD

**Discussion by DesireWithin** [StackStorm/st2#5406] [Q&A]
Hello,
I've been having some trouble lately,
I have a `Jenkins.build_job_enh` task, input parameter is a list, it will simultaneously triggers the Jenkins job.
How ever, if Jenkins job timeout( I mean Stackstorm task timeout) or failed, the result of task is only a string `None`. I can't get useful information such as Jenkins Job id or parameters for this particular task.
This means that stackstorm cannot automatically handle errors, can't eliminate false results from correct ones.
`taks(task_id)` so far only include the following values:
- task_execution_id
- workflow_execution_id
- task_name
- task_id
- route
- result
- status
- start_timestamp
- end_timestamp

Why don't we add input paramter as well?
I think it is very convenient for handling  `Orquesta  With Items Model`
Currently my workround is modify Jenkins Pack code, let it return input paramters. It's useful for the Jenkins Job failed, but not for stackstorm task timeout.

**Comment by minsis** [comment]
Have you tried going to the Jenkins pack github page and either opening an issue there or contributing what code changes you made to it?

https://github.com/StackStorm-Exchange/stackstorm-jenkins
